build.gradle

```diff
@@ -108,7 +108,7 @@ subprojects {
         }
 
         testLogging.showStandardStreams = true
-        testLogging.exceptionFormat = 'full'
+        testLogging.exceptionFormat = 'short'
```

Comment:
Try to print shorter stack traces in the tests

Reply:
By doing a `println(testLogging.exceptionFormat)` it seems to me that this format is already 'short',
so I guess we can directly remove the line, if I'm not missing something